truffle.js

```diff
@@ -5,5 +5,11 @@ module.exports = {
       port: 8545,
       network_id: "*" // Match any network id
     }
+  },
+  solc: {
+    optimizer: {
+      enabled: true,
+      runs: 200
+    }
   }
 };
```

Comment:
should we add little comment on optimizer specifically on runs: 200?

Reply:
will add